src/pybind/mgr/nfs/export.py

```diff
@@ -44,6 +45,8 @@
 
 log = logging.getLogger(__name__)
 
+cephfs = None   # holds CephFS Python bindings
```

Comment:
Are you sure the cephfs python bindings do not need any initialization? How does this module know which file system to mount?

Reply:
I think it should work, i see similar usage in cephfs-shell too. Although I'll try dig into this and come up with a better explanation.

Reply:
You are right, it needs initialization, make check failed due to this:

```
Traceback (most recent call last):
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/nfs/export.py", line 400, in create_export
    return self.create_cephfs_export(**kwargs)
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/nfs/export.py", line 643, in create_cephfs_export
    if not cephfs.is_dir_exists(path):
AttributeError: 'NoneType' object has no attribute 'is_dir_exists'```